lite2/client.go

```diff
@@ -3,12 +3,11 @@ package lite
 import (
 	"bytes"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
could you run`make format`

Reply:
```
callum@CallumsPC:~/go/src/github.com/tendermint/tendermint$ make format
find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/tendermint/tendermint
xargs: goimports: No such file or directory
Makefile:139: recipe for target 'format' failed
make: *** [format] Error 127
```

Reply:
odd, do you have goimports installed locally?